homeassistant/components/device_automation/trigger.py

```diff
@@ -37,7 +37,7 @@ async def async_get_trigger_capabilities(
     ) -> dict[str, vol.Schema]:
         """List trigger capabilities."""
 
-    async def async_get_triggers(
+    async def async_get_triggers(  # type: ignore[override]
```

Comment:
Added to silence mypy. In real, I think `DeviceAutomationTriggerProtocol` should not inherit `TriggerProtocol`, they are not related.

Reply:
Handled by https://github.com/home-assistant/core/pull/144888, when that PR is merged the mypy directive can be removed

Reply:
This is no longer relevant, #144888 has been merged and the mypy override has been removed